src/features/document/document_feature.py

```diff
@@ -11,12 +11,18 @@
 from .use_cases.add_document_to_path_use_case import add_document_use_case
 from .use_cases.add_raw_use_case import add_raw_use_case
 from .use_cases.get_document_use_case import get_document_use_case
-from .use_cases.remove_by_path_use_case import remove_by_path_use_case
-from .use_cases.remove_use_case import remove_use_case
+from .use_cases.remove import remove_use_case
 from .use_cases.update_document_use_case import update_document_use_case
 
 router = APIRouter(tags=["default", "document"], prefix="/documents")
 
+"""    - **reference**: A reference to a package or a data source
+      - By id: PROTOCOL://DATA SOURCE/$ID.Attribute
+      - By path: PROTOCOL://DATA SOURCE/ROOT PACKAGE/SUB PACKAGE/ENTITY.Attribute
+      - By query: PROTOCOL://DATA SOURCE/$ID.list(key=value)
+
+      The PROTOCOL is optional, and the default is dmss."""
```

Comment:
Looks like this comment about reference is moved out from function docstring. The advantage with keeping it in the docstring is that it will be included in the swagger documentation  - I would prefer to move it back into the function docstring